src/vs/workbench/browser/parts/views/treeView.ts

```diff
@@ -1442,9 +1440,9 @@ export class CustomTreeViewDragAndDrop implements ITreeDragAndDrop<ITreeItem> {
 
 	onDragOver(data: IDragAndDropData, targetElement: ITreeItem, targetIndex: number, originalEvent: DragEvent): boolean | ITreeDragOverReaction {
 		const types: Set<string> = new Set();
-		originalEvent.dataTransfer?.types.forEach((value, index) => {
-			if (INTERNAL_MIME_TYPES.indexOf(value) < 0) {
-				types.add(this.convertKnownMimes(value).type);
+		originalEvent.dataTransfer?.types.forEach((type) => {
+			if (INTERNAL_DND_MIME_TYPES.indexOf(type) < 0) {
+				types.add(type === DataTransfers.RESOURCES.toLowerCase() ? Mimes.uriList : type);
```

Comment:
This also doesn't properly convert `Files` to `text/uri-list`. `Files` is the mime type used when files are dragged from the OS.